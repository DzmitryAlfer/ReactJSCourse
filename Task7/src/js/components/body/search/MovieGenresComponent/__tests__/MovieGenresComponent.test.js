import React from 'react';
import Renderer from 'react-test-renderer';
import { MovieGenresComponent } from '../index';

describe('MovieGenresComponent', () => {
    test('Component render', () => {
        const tree = Renderer.create(<MovieGenresComponent genres={['genre1', 'genre2', 'genre3']} />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
