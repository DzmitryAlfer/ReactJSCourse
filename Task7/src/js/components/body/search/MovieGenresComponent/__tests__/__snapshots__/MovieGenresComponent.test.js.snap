// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieGenresComponent Component render 1`] = `
<div
  className="movie-genres"
>
  <span
    className="movie-genres_item"
  >
    genre1
  </span>
  <span
    className="movie-genres_item"
  >
    genre2
  </span>
  <span
    className="movie-genres_item"
  >
    genre3
  </span>
</div>
`;
